Fundamentals/Data/Scheduled-Publishing/index.md

```diff
@@ -58,4 +58,4 @@ If your scheduled publishing/unpublishing is not working as you would expect it
 
 To better diagnose the issue you can temporarily change your log4net config settings to be DEBUG instead of INFO. This will give you all sorts of information including being able to see whether or not the scheduled publishing endpoint is being reached or not.
 
-In some cases it might be easiest to specify the [umbracoSettings:settings/web.routing/@umbracoApplicationUrl](../../../Reference/Config/umbracoSettings/index-v8.md#web-routing) setting to ensure that your server is communicating to itself on the correct base url.
+In some cases it might be easiest to specify the [umbracoSettings:settings/web.routing/@umbracoApplicationUrl](../../../Reference/Config/umbracoSettings/index.md#web-routing) setting to ensure that your server is communicating to itself on the correct base url.
```

Comment:
This is linking to a v9 doc.

Is there a v10 equivalent we can link to as well?

Reply:
I am not sure. Looking at https://our.umbraco.com/documentation/Reference/V9-Config/umbracoSettings/#webrouting seems like it has been reverted from v9.00 to 8.00-8.17.0 (3 days back). If this is valid only for v8 it should be moved to the v8 config folder.

Hopefully, there is a new article (in progress) for v9/10 or the info is added somewhere else.

Reply:
Yes, that article needs to be removed. There is no `umbracoSettings` file in v9 or v10.

I believe this would be the correct link for v9+: https://our.umbraco.com/documentation/Reference/V9-Config/WebRoutingSettings/

Reply:
Sounds good. Updated the link.